google-cloud-datastore/lib/google/cloud/datastore/v1/datastore_client.rb

```diff
@@ -318,14 +318,11 @@ def lookup \
           #
           #   # TODO: Initialize `project_id`:
           #   project_id = ''
-          #
-          #   # TODO: Initialize `partition_id`:
-          #   partition_id = {}
-          #   response = datastore_client.run_query(project_id, partition_id)
+          #   response = datastore_client.run_query(project_id)
 
           def run_query \
               project_id,
-              partition_id,
+              partition_id: nil,
```

Comment:
@dazuma Can you double-check and if we should accept this change of `partition_id` to a keyword arg, and if so, approve this PR?

Reply:
Datastore changed to gapic v2. Hold off on merging while I check whether these changes were intentional or due to missed annotations.

Reply:
Okay, it looks like the changes were intentional. Since this is the low-level client, I'm inclined to take the breaking change.